From eb3bd2c1ed8b1b65851bb4274b1ab5c20baeb3fa Mon Sep 17 00:00:00 2001
From: Duan Jiong <duanj.fnst@cn.fujitsu.com>
Date: Fri, 11 Apr 2014 16:38:12 +0800
Subject: [PATCH 113/213] rbd: replace IS_ERR and PTR_ERR with PTR_ERR_OR_ZERO
References: fate#318918
Git-commit: 461f758ac0bad40fe8e0959f415dae38efa16c12
Patch-mainline: v3.16-rc1

This patch fixes coccinelle error regarding usage of IS_ERR and
PTR_ERR instead of PTR_ERR_OR_ZERO.

Signed-off-by: Duan Jiong <duanj.fnst@cn.fujitsu.com>
Reviewed-by: Yan, Zheng <zheng.z.yan@intel.com>
Acked-by: David Disseldorp <ddiss@suse.de>

---
 drivers/block/rbd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/block/rbd.c b/drivers/block/rbd.c
index 4f910e5..4421e8b 100644
--- a/drivers/block/rbd.c
+++ b/drivers/block/rbd.c
@@ -4822,7 +4822,7 @@ static int rbd_dev_image_id(struct rbd_device *rbd_dev)
 
 		image_id = ceph_extract_encoded_string(&p, p + ret,
 						NULL, GFP_NOIO);
-		ret = IS_ERR(image_id) ? PTR_ERR(image_id) : 0;
+		ret = PTR_ERR_OR_ZERO(image_id);
 		if (!ret)
 			rbd_dev->image_format = 2;
 	} else {
-- 
2.1.4

